WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -1652,6 +1652,16 @@ extension ReaderStreamViewController : WPTableViewHandlerDelegate {
                 syncHelper.syncMoreContent()
             }
         }
+
+        // Bump the render tracker if necessary.
+        let posts = tableViewHandler.resultsController.fetchedObjects as! [ReaderPost]
+        let post = posts[indexPath.row]
+        let railcar = post.railcarDictionary()
+        if post.isKindOfClass(ReaderGapMarker) || railcar == nil || post.rendered {
+            return
+        }
+        post.rendered = true
+        WPAppAnalytics.track(.TrainTracksRender, withProperties: railcar)
```

Comment:
I can't tell for certain, but it looks like `.TrainTracksRender` is being called multiple times for cells/posts after paging has occurred. If you run breakpoints or logs you can see subsequent calls made for the same posts.